packages/fast-components-styles-msft/src/select/index.ts

```diff
@@ -13,44 +17,57 @@ import {
 } from "../utilities/color";
 import { applyFloatingCornerRadius } from "../utilities/border";
 import { designUnit } from "../utilities/design-system";
-
-export const selectDisplayButtonOverrides: ComponentStyles<
-    Partial<ButtonClassNameContract>,
-    DesignSystem
-> = {
-    button: {
-        width: "100%",
-        padding: "0 10px",
-    },
-    button_contentRegion: {
-        width: "100%",
-        display: "inline-flex",
-    },
-};
+import { inputFieldStyles } from "../patterns/input-field";
 
 const styles: ComponentStyles<SelectClassNameContract, DesignSystem> = (
     config: DesignSystem
 ): ComponentStyleSheet<SelectClassNameContract, DesignSystem> => {
     const designSystem: DesignSystem = withDesignSystemDefaults(config);
+    const direction: Direction = designSystem.direction;
 
     return {
         select: {
             minWidth: "276px",
             maxWidth: "374px",
         },
 
+        select_button: {
+            height: height(),
+            width: "100%",
+            ...inputFieldStyles(),
+        },
+
+        select_button_contentRegion: {
+            display: "grid",
+            gridTemplateColumns: "1fr auto",
+            alignItems: "center",
+            justifyItems: "start",
+        },
+
+        select_button_displayText: {
+            ...ellipsis(),
+            textAlign: applyLocalizedProperty("left", "right", direction),
+            width: "100%",
+        },
+
         select_toggleGlyph: {
+            [applyLocalizedProperty(
+                "margin-left",
+                "margin-right",
+                direction
+            )]: horizontalSpacing(),
             fill: neutralForegroundRest,
+            width: glyphSize,
+            height: glyphSize,
+            gridColumnStart: "2",
         },
 
-        select__disabled: {
-            opacity: ".3",
-        },
+        select__disabled: {},
 
         select_menu: {
             background: neutralFillStealthRest,
             ...applyFloatingCornerRadius(),
-            ...elevation(ElevationMultiplier.e11)(designSystem),
+            ...elevation(ElevationMultiplier.e11)(withDesignSystemDefaults(config)),
```

Comment:
I think you can use the new `...applyElevation` here and avoid direct references to the `designSystem`

Reply:
Yes, got that from the merge with Brian's changes, but still need designSystem to get direction.